lib/filters/index.js

```diff
@@ -1,6 +1,7 @@
 const minimatch = require('minimatch');
 const pathRegexp = require('path-to-regexp');
 const qs = require('qs');
+const path = require('path');
```

Comment:
Could get a smaller diff by just importing this differently, e.g. `const pathLib = require('path');`

Reply:
I went with this option because I prefer to call packages by their 'real' name as I just find that easier to grok.

Reply:
> I went with this option because I prefer to call packages by their 'real' name

That means you let other people decide your variable names :D

> I just find that easier to grok

Doesn't your editor `grok` stuff for you? Anyway, each to their own :D